www/css/app.css

```diff
@@ -75,6 +75,7 @@
     padding-left: 12px;
     padding-right: 12px;
     background: lightblue;
+    z-index: 1;
```

Comment:
I don't see how this relates to this PR

Reply:
@mossroy It turns out that the Article List (search results) have the wrong z-index, which is why the last search result in a long list can sometimes get hidden under the bottom bar. When the screen is inverted, this has the effect that the search results are hidden (they are no longer visible at all, though they can still be selected with the keyboard). The cure to both issues is to correct the z-index of the Article List. So it does need to be in this PR, even though it is a wider issue that this PR has revealed.